tensorbay/opendataset/HKD/__init__.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python3
```

Comment:
the "first commit" in commit message is meaningless,
check out the commit messages for other data loaders in this repo for reference.